src/EventListener/LoginListener.php

```diff
@@ -34,25 +30,12 @@ public function onLogin(LoginEvent $event): void
     }
 
     private function handleLogin(User $user): void
-    {
-        // update password hash if necessary
```

Comment:
Don't you think we'll need this functionality in the future?

Reply:
No, I don't see it for the future, to be honest.

Reply:
But how do you think, we'll do updates to future hashing algorithms?

Reply:
We decide and implement it when it is necessary. At the moment it is dead code.